prisma/schema.prisma

```diff
@@ -13,6 +13,8 @@ model User {
   username      String     @unique
   email         String     @unique
   hash          String
+  userClass     String?
```

Comment:
Nitpick: Traditionally fields in a Postgres database are named with snake_case. This is why the fields createdAt and updatedAt have mappings to their column name.

Reply:
Gotcha that makes sense why i was having difficulty lol lesson learned